src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -100,6 +100,50 @@ public GHCreateRepositoryBuilder createRepository(String name) {
         return new GHCreateRepositoryBuilder(root, "/orgs/" + login + "/repos", name);
     }
 
+    /**
+     * Starts a builder that creates a new repository.
+     *
+     * <p>
+     * You use the returned builder to set various properties, then call {@link GHCreateRepositoryBuilder#create()} to
+     * finally create a repository.
+     *
+     * @param name
+     *            the name
+     * @param isTemplate
+     *            the is_template
+     * @return the gh create repository builder
+     */
+    public GHCreateRepositoryBuilder createRepositoryWithParametersIsTemplate(String name, Boolean isTemplate) {
+        return new GHCreateRepositoryBuilder(root, "/orgs/" + login + "/repos", name, isTemplate);
+    }
+
+    /**
+     * Starts a builder that creates a new repository.
+     *
+     * <p>
+     * You use the returned builder to set various properties, then call
+     * {@link GHCreateRepositoryBuilder#createWithTemplate()} to finally create a repository.
+     *
+     * @param templateOwner
+     *            the owner of template repository
+     * @param templateRepo
+     *            the template repository
+     * @param name
+     *            the name of repository to be created
+     * @param owner
+     *            the organization of repository to be created
+     * @return the gh create repository builder
+     */
+    public GHCreateRepositoryBuilder createRepositoryWithTemplate(String templateRepo,
```

Comment:
Ah, I see.  The create from template repository has almost none of the parameters of general repository creation.  
Still I think that limitation is something that could be explained in the javadoc for `fromTemplateRepository(String templateRepo, String templateOwner)` and include a link to the GitHub docs.  

What you've done is good: guiding users to the a non-failure path.  However, in this case, I'd rather reduce the surface area and added code in this library.  

Also add `@Preview` and `@Deprecated` annotations to the methods that call GitHub Preview APIs.

Reply:
“Also add @Preview and @Deprecated annotations to the methods that call GitHub Preview APIs.” —— I see. Thank you for reminding me. But I don't know why need to add the @Deprecated annotation.

Reply:
> “Also add @Preview and @deprecated annotations to the methods that call GitHub Preview APIs.” —— I see. Thank you for reminding me. But I don't know why need to add the @deprecated annotation.

I found the answer from oracle official doc:
When to Deprecate
When you design an API, carefully consider whether it supersedes an old API. If it does, and you wish to encourage developers (users of the API) to migrate to the new API, then deprecate the old API. Valid reasons to deprecate an API include:
* It is insecure, buggy, or highly inefficient (I got  this point)
* It is going away in a future release
* It encourages bad coding practices